packages/enzyme/src/selectors.js

```diff
@@ -239,8 +239,17 @@ function isComplexSelector(tokens) {
  * @param {Function|Object|String} selector
  */
 export function buildPredicate(selector) {
-  // If the selector is a function, check if the node's constructor matches
-  if (typeof selector === 'function') {
+  // If the selector is a string, parse it as a simple CSS selector
+  if (typeof selector === 'string') {
```

Comment:
these changes also seem like they could be peeled out into a separate PR?

Reply:
They could yeah, I wanted to validate that the apis I was adding were sufficient